sample_project/sample_project/app/v1/books/service.py

```diff
@@ -0,0 +1,92 @@
+from flask import jsonify
+
+from app.connections import SqlConnection
+class SubApp1Service:
+    db_config = {
+        'host': 'localhost',
+        'user': 'root',
+        'password': 'root',
+        'database': 'demo'
+    }
+
+    db_connection = SqlConnection(db_config)
+
+    def __init__(self, params, headers):
+        self.params = params
+        self.headers = headers
+
+    def get_static_api_response(self):
+        return self.params, 'success'
+
+    def get_books(self):
+        try:
+            query = "select * from books"
```

Comment:
All queries will be done in the coordinator only logic in service file.

Reply:
ok